tests/util/kube_utils.go

```diff
@@ -611,3 +616,70 @@ func GetKubeConfig(filename string) error {
 	log.Infof("kubeconfig file %s created\n", filename)
 	return nil
 }
+
+// CreateMultiClusterSecrets will create the secrets and configmap associated with the remote cluster
+func CreateMultiClusterSecrets(namespace string, KubeClient kubernetes.Interface, RemoteKubeConfig string) error {
+	const (
+		secretName    = "remote-cluster"
```

Comment:
have you verified this secret name works?  the secret name  should match the cluster name.  I'm not sure if the remote-cluster is called "remote-cluster" or some other name.

Reply:
Yes I have tested this - I choose not to name everything with a ".yaml"extension.